### PR TITLE
Output solidity ABI file

### PR DESCRIPTION
**Pull request by swapnilraj**
Don't mangle output file paths!!
storage code moved below

In a future PR we can remove abiExtractor as well

### CODE REVIEW

**Review by JorikSchellekens — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -134,9 +134,9 @@ solc-v0.7.6
 
 # output from warp
 warp_output/
-example__contracts
-example_contracts/**/*.cairo
-example_contracts/**/*.json
+warp_test/
+warp_test/**/*.cairo
```
Comment (JorikSchellekens) [inline .gitignore:138]: The two globed paths are not required

File: src/testing.ts [hunk 2]
```diff
@@ -257,10 +256,12 @@ export function runTests(force: boolean, onlyResults: boolean, unsafe = false, e
   if (force) {
     postTestCleanup();
   } else if (!preTestChecks()) return;
+  console.log('findSolSourceFilePaths');
```
Comment (JorikSchellekens) [inline src/testing.ts:259]: Console log

File: src/testing.ts [hunk 3]
```diff
@@ -276,12 +277,12 @@ export function runTests(force: boolean, onlyResults: boolean, unsafe = false, e
 }
 
 function preTestChecks(): boolean {
-  if (!checkNoCairo('example__contracts')) {
-    console.log('Please remove example__contracts, or run with -f to delete it');
+  if (!checkNoCairo(path.join('warp_test', 'example_contracts'))) {
+    console.log('Please remove example_contracts, or run with -f to delete it');
```
Comment (JorikSchellekens) [inline src/testing.ts:281]: Should probably read `warp_test/example_contracts` instead

**Review by JorikSchellekens — CHANGES_REQUESTED** [review]

File: src/testing.ts (hunk 2 above)
Comment (swapnilraj) [reply src/testing.ts:259]: oopsie

File: src/testing.ts (hunk 3 above)
Comment (swapnilraj) [reply src/testing.ts:281]: why?

File: src/testing.ts (hunk 2 above)
Comment (swapnilraj) [reply src/testing.ts]: fixed

File: .gitignore (hunk 1 above)
Comment (swapnilraj) [reply .gitignore]: fixed

File: src/testing.ts (hunk 3 above)
Comment (JorikSchellekens) [reply src/testing.ts:279]: Because that's the folder that should be removed

Comment (swapnilraj) [reply src/testing.ts]: right I see, I was confused between the code and string literal

**Review by JorikSchellekens — APPROVED** [review]

File: src/passes/filePathMangler.ts [hunk 4]
```diff
@@ -1,15 +1,11 @@
-import { ImportDirective, SourceUnit } from 'solc-typed-ast';
```
Comment (rodrigo-pino) [inline src/passes/filePathMangler.ts:1]: I think we can safely delete this pass now, and move the checkPath to rejectUnsupportedFeatures

**Pull request merged**